gwcs/tests/test_wcs.py

```diff
@@ -486,3 +486,20 @@ def test_pixel_to_world(self):
         assert isinstance(sky_coord, coord.SkyCoord)
         assert_allclose(sky_coord.data.lon.value, ra)
         assert_allclose(sky_coord.data.lat.value, dec)
+
+
+def test_to_fits_sip():
+    y, x = np.mgrid[:1024:10, :1024:10]
+    xflat = np.ravel(x[1:-1, 1:-1])
+    yflat = np.ravel(y[1:-1, 1:-1])
+    warg = np.stack((xflat, yflat)).transpose()
+    af = asdf.open('data/miriwcs.asdf')
+    miriwcs = af.tree['wcs']
+    bounding_box = ((0, 1024), (0, 1024))
+    mirisip = miriwcs.to_fits_sip(bounding_box, max_error=1.e-10, max_inv_error=0.1)
+    fitssip = astwcs.WCS(mirisip)
+    fitsvals = fitssip.all_pix2world(warg, 1)
+    assert_allclose(miriwcs(xflat, yflat),
+                    fitsvals.transpose(), atol=10**-10)
```

Comment:
I do not think this test guarantees a good agreement. For typical RA and DEC angles (~tens or hundreds), default `rtol` of `1e-9` would become equivalent to `atol` of 1e-4 - 1e-5 which is a very high error (in "cosmic" terms).

By the way (and out of curiosity), why do you write `10**-10` and not `1e-10`?

Reply:
I was coloring my Fortran textbook and that was the last example I saw.

Not sure what you mean about the high error. In terms of degrees, 1e-10  is pretty small. That is the unit of the values here, right?

Reply:
From the [documentation of `assert_allclose`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_allclose.html):

> It compares the difference between _actual_ and _desired_ to `atol + rtol * abs(desired)`

So, it does not matter that you set `atol=1e-100` because the `rtol * abs(angle)` will dominate and so, practically for an angle of 150 degrees, `1e-7 * abs(150)` is `1.5e-5` which is a huge error.

Reply:
OK, I set rtol to 0